src/item.cpp

```diff
@@ -2934,7 +2952,14 @@ std::string item::display_name( unsigned int quantity ) const
         }
     }
 
-    return string_format( "%s%s%s", name.c_str(), sidetxt.c_str(), amt.c_str() );
+    std::string &prefixed_name = name;
```

Comment:
Why that? Can't you just use the variable `name` directly?

Reply:
Right, I thought `name` was `item` field and didn't want to change that.